package.json

```diff
@@ -8,7 +8,7 @@
     "clean": "rm -rf dist",
     "lint": "tslint --project .",
     "test": "IASQL_ENV=test jest -i",
-    "coverage": "IASQL_ENV=test jest -i --coverage --forceExit",
+    "coverage": "IASQL_ENV=test jest -i --coverage --forceExit --bail",
```

Comment:
Added this to make the tests fail on the first error and do not wait until all the suite is done to see if something failed. I believed it would help us, but we can revert if we do not like it

Reply:
ohhh ok, so `--bail` work between suites, but here we want to stop tests within the same suite (https://github.com/facebook/jest/issues/2626). The solution for us should go in something like this https://thewebdev.info/2022/02/24/how-to-stop-test-suite-after-first-fail-with-jest/

I'll remove that option